exec/compressible_mui/main_driver.cpp

```diff
@@ -620,15 +636,23 @@ void main_driver(const char* argv)
             structFactCons.FortStructure(structFactConsMF,geom);
             if(project_dir >= 0) {
                 MultiFab primFlattened;  // flattened multifab defined below
+                MultiFab consFlattened;  // flattened multifab defined below
                 if (slicepoint < 0) {
                     ComputeVerticalAverage(prim, primFlattened, geom, project_dir, 0, structVarsPrim);
```

Comment:
Also, this should be ComputeVerticalAverage(structFactPrimMF, primFlattened, geom, project_dir, 0, structVarsPrim); instead of ComputeVerticalAverage(prim, primFlattened, geom, project_dir, 0, structVarsPrim);

and so on, everywhere.